core-blocks/archives/index.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+
+export const name = 'core/archives';
+
+export const settings = {
+	title: __( 'Archives' ),
+
+	description: __( 'Display a monthly archive of your site’s Posts.' ),
```

Comment:
FWIW, filter `widget_archives_dropdown_args` can change the periodic interval, meaning "monthly" may become inaccurate. However, I could see filtering as a more advanced feature, and sticking with "monthly" offers the benefit of conveying the basic idea of the block more clearly than e.g. "Display a periodic archive […]". Because of this, I don't mind sticking with the current copy.